### PR TITLE
fix: [SRM-14968]: Pass queryParams while routing

### PR DESCRIPTION
**Pull request by deepesh-ui**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/__tests__/ErrortrakingRouteChange.test.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      projectIdentifier: 'project1'
+    }}
+    queryParams={queryParams}
+  >
+    <RedirectToCETEventSummaryDetail />
+  </TestWrapper>
+)
+describe('Validate Error tracking old notificaiton lik', () => {
+  test('should redirect to new url', () => {
+    expect(1).toEqual(1)
```
Comment (praneshg239) [inline src/modules/85-cv/__tests__/ErrortrakingRouteChange.test.tsx:36]: please remove this

File: src/modules/85-cv/__tests__/ErrortrakingRouteChange.test.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    const { container } = render(<Wrapper />)
+    const location = container.querySelector('[data-testid="location"]')?.textContent
+    expect(location).toEqual('/account/cvng/cet/orgs/default/projects/project1/eventsummary/events?accountId=cvng')
+  })
+
+  test('should redirect to new url with queryParams', () => {
+    expect(1).toEqual(1)
```
Comment (praneshg239) [inline src/modules/85-cv/__tests__/ErrortrakingRouteChange.test.tsx:43]: same here

**Review by praneshg239 — APPROVED** [review]

File: src/modules/85-cv/__tests__/ErrortrakingRouteChange.test.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      accountId: 'cvng',
+      orgIdentifier: 'default',
+      projectIdentifier: 'project1'
+    }}
+    queryParams={queryParams}
+  >
+    <RedirectToCETEventSummaryDetail />
+  </TestWrapper>
+)
+describe('Validate Error tracking old notificaiton lik', () => {
```
Comment (guycastel) [inline src/modules/85-cv/__tests__/ErrortrakingRouteChange.test.tsx:34]: typo: lik -> link

File: src/modules/85-cv/__tests__/ErrortrakingRouteChange.test.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2023 Harness Inc. All rights reserved.
```
Comment (guycastel) [inline src/modules/85-cv/__tests__/ErrortrakingRouteChange.test.tsx:2]: typo in filename: ErrortrakingRouteChange -> ErrorTrackingRouteChange

**Review by karansaraswat19 — APPROVED** [review]

**Review by guycastel — APPROVED** [review]

**Review by ishantmehtaharness — APPROVED** [review]
Looks Good to me

**Pull request closed without merging**